scalafmt-core/shared/src/main/scala/org/scalafmt/util/PolicyOps.scala

```diff
@@ -8,22 +8,20 @@ import org.scalafmt.internal.Policy
 
 object PolicyOps {
 
-  class PenalizeAllNewlines private[PolicyOps] (f: Policy.Pf, endPos: Int)(
-      implicit line: sourcecode.Line
-  ) extends Policy.Clause(f, endPos)
-
-  def penalizeAllNewlines(
+  case class penalizeAllNewlines(
```

Comment:
nit
```suggestion
  case class PenalizeAllNewlines(
```

Reply:
will do. was trying to avoid renaming about 15 invocations in router, as they were all using the method.